cluster-controller/src/main/java/io/strimzi/controller/cluster/operator/resource/StatefulSetOperator.java

```diff
@@ -196,10 +197,10 @@ public void eventReceived(Action action, Pod pod) {
         @Override
         public void onClose(KubernetesClientException e) {
             if (e != null && !deleted.isComplete()) {
-                log.error("Kubernetes watcher has been closed with exception!", e);
+                log.error("Rolling update watcher has been closed with exception!", e);
```

Comment:
Do we really need the `!`s?

Reply:
I like them ! :-)

Reply:
I removed them. I didn't even noticed them.